src/main/scala/BintrayKeys.scala

```diff
@@ -0,0 +1,84 @@
+package bintray
+
+import sbt._
+
+trait BintrayKeys {
+  val bintray = taskKey[String]("bintray-sbt is an interface for the bintray package service")
+
+  val bintrayRelease = taskKey[Unit](
+    "Releases a version of package on bintray")
+
+  val bintrayReleaseOnPublish = settingKey[Boolean](
+    "When set to true, publish also runs bintrayRelease.")
+
+  val bintrayOrganization = settingKey[Option[String]](
+    "Bintray organization name to publish to. Defaults to None unless project is an sbtPlugin")
+
+  val bintrayPackage = settingKey[String](
+    "Bintray package name")
+
+  val bintrayRepository = settingKey[String](
```

Comment:
I see all of these key names got at least 7 characters longer. Is the new plugin recommendation to have longer, prefixed key names to avoid collisions rather than name spacing on a task axis?